src/docs/metrics/active-deposits/index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Active Deposits
+author: Santiment Team
+date: 2020-04-06
+
+headline: "Crypto Active Addresses: Metric to Track Blockchain Activity"
+description: "The active addresses metric is the number of unique addresses participating in transactions during one day. Learn more about how we calculate it."
+
+---
+
+## Definition
+
+### Active Deposits
+
+Shows the **number of unique deposit addresses** that have been active on a
+particular day for a given asset. Refer to a [deposit address explanation here](/labels/deposit)
+
+![ethereum daily active deposits](daily-active-deposits-ethereum.png)
+
+### Deposit Transactions
+
+Deposit transactions metric shows the total number of all **incoming and outcoming transactions** involving
+deposit addresses on a particular day.
+
+As such, this metric accounts for both user-to-exchange transactions:
+
+1. from a personal wallet to a deposit address, and
+2. from a deposit address to the main exchange wallet.
+
+Deposit-related Transactions is often valuable when used in conjunction with
+**Exchange Inflow** -- [another one of our
+metrics](/metrics/exchange-flow-metrics) -- which shows the total amount of
+coins moving to exchange wallets on a particular day.
+
+So, for example, an **increase in exchange inflow** but a **plateauing number of
+deposit-related transactions** means that more coins are entering the exchanges,
+although the number of transactions to the exchanges stayed the same. In other
+words, **the average deposit size has increased.**
+
+For an exchange-specific data please refer to [that page](/metrics/labeled-exchange)
```

Comment:
Let's also mention the metric names here.
We can also here make the distinction between no suffix and `_5m` suffix.